src/database/index.tsx

```diff
@@ -540,6 +540,20 @@ export const removeScenarioUser = async (
   }
 };
 
+const batchSetFacilityModelVersions = (
+  modelVersions: ModelInputs[],
+  facilityDocRef: firebase.firestore.DocumentReference,
+  batch: firebase.firestore.WriteBatch,
+) => {
+  const modelVersionDoc = facilityDocRef
+    .collection(modelVersionCollectionId)
+    .doc();
```

Comment:
Sorry, I gave you bad advice.  I see now that this is creating a new modelVersion doc for each model version, which is desired.  With this outside of the loop, we end up only creating a single new modelVersion with whatever the last values are in the modelVersions array.

Reply:
Oooh, yeah makes sense! sorry I didn't look at it closely when updating...will change it back